docs/pages/installation/helm.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Helm Installation
+permalink: installation/helm
+type: installation
+display-title: "true"
+language: en
+list: exclude
+---
+
+<div class="prereqs"><p><strong style="font-size: 20px;">Prerequisites</strong> </p> 
+  <ol>
+    <li><a href="https://helm.sh/docs/intro/install/" class="meshery-light">Helm</a>  should be installed on your local machine.</li>
+    <li>You should have access to the cluster/platform where you want to deploy Meshery.</li>
+    <li>Ensure that the kubeconfig file has the correct current context/cluster configuration.</li>
+  </ol>
+</div> 
+
+## Install Meshery on Your Kubernetes Cluster Using Helm V3
+
+We strongly recommend using Helm v3 because this version no longer includes the Tiller component, making it lighter and safer.
```

Comment:
We can safely remove this statement and just assume v3. 

We can also remove the `v3` in the heading. No one should be using v2 anymore, and if they are, that's their own issue - not for us to worry about.

Reply:
sure will make this change and commit again